# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/angle/checkout/out/gen/angle/id/commit.h
# Commit: 13282d7a47a5
# Full Hash: 13282d7a47a57a73be8d060efd9645b4d642f03e
# Author: Miko Mynttinen <mikokm@gmail.com>
# Date: 2019-09-08 09:46:52
# Regressor Bug: 1578576
# File Overlap Count: 4
# Description:
#   Bug 1578576 - Part 2: Update to ANGLE 3865 r=jgilbert
#   
#   Depends on D44561
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D44579
# ==============================================================================

diff -r 3064469c073d -r 13282d7a47a5 gfx/angle/checkout/out/gen/angle/id/commit.h
--- a/gfx/angle/checkout/out/gen/angle/id/commit.h	Sat Sep 07 13:36:46 2019 +0000
+++ b/gfx/angle/checkout/out/gen/angle/id/commit.h	Sat Sep 07 13:38:36 2019 +0000
@@ -1,3 +1,3 @@
-#define ANGLE_COMMIT_HASH "21dccd386996"
+#define ANGLE_COMMIT_HASH "0f5a52939b91"
 #define ANGLE_COMMIT_HASH_SIZE 12
-#define ANGLE_COMMIT_DATE "2019-08-01 14:11:16 -0700"
+#define ANGLE_COMMIT_DATE "2019-09-03 21:51:25 +0200"